pkg/services/app/add.go

```diff
@@ -135,7 +136,7 @@ func (a *App) Add(params AddParams) error {
 
 	a.printAddSummary(params)
 
-	if err := IsClusterReady(a.Logger, a.Kube); err != nil {
+	if err := IsClusterReady(&loggerfakes.FakeLogger{}, a.Kube); err != nil {
```

Comment:
Is the fake logger here intentional?

Reply:
Yes, because we now check the status up front when we set up the git clients so we were getting all the status info printed out twice. I didn't want to entirely remove the check in `add` for when it gets used by the UI, so I passed in a fake logger.